src/kirby/components/button/button.component.tns.global.scss

```diff
@@ -1,23 +1,19 @@
 @import '../../scss/native-imports';
 
 Button[kirby-button] {
-  background-color: white;
-  border-color: lightskyblue;
+  color: #{get-color(map-get($themes, 'dark'), 'on-primary')};
```

Comment:
Tror det bliver træls (For at bruge det gode jyske ord) at skulle skrive det der hver gang man skal have en farve. Men forstår godt at du gerne vil undgå at skulle angive declarationen ligesom før - kunne vi evt. lave en function der tager farver fra $themes og måske bare angiver default til dark hvis andet ikke er angivet.

Noget ala: 
color: get-color('on-primary');
og 
color: get-color('on-primary', 'some-other-theme');